client/src/Layout/useStyles.ts

```diff
@@ -26,6 +26,9 @@ const useStyles = makeStyles((theme) => ({
     textDecoration: 'underline',
     textTransform: 'uppercase',
   },
+  textWhite: {
```

Comment:
Optional: If this is a one time occurrence it is okay but ideally it would be better to add colors within the MUI theme and use the theme to apply it.